.pre-commit-config.yaml

```diff
@@ -93,7 +93,7 @@ repos:
     rev: v1.0.0
```

Comment:
Could we include , syft-cli also in `black` pre-commit hook, line no: 73 in .pre-commit-config.yaml
under `files`

Reply:
Please see this comment - https://github.com/OpenMined/PySyft/pull/7997#issuecomment-1678621326